app/lib/methods/getRoles.js

```diff
@@ -2,9 +2,20 @@ import { sanitizedRaw } from '@nozbe/watermelondb/RawRecord';
 
 import database from '../database';
 import log from '../../utils/log';
+import reduxStore from '../createStore';
 import protectedFunction from './helpers/protectedFunction';
+import { setRoles as setRolessAction } from '../../actions/roles';
 
-export default function() {
+export async function setRoles() {
+	const db = database.active;
+	const rolesCollection = db.collections.get('roles');
+	const allRoles = await rolesCollection.query().fetch();
+	const parsed = allRoles.reduce((acc, item) => ({ ...acc, [item.id]: item.description }), {});
```

Comment:
If description is empty, can you add the id as the description?
Role's descriptions aren't required on admin panel, but they might exist anyway.
This is going to prevent roles from not showing.
Something like this:

```suggestion
	const parsed = allRoles.reduce((acc, item) => ({ ...acc, [item.id]: item.description || item.id }), {});
```